server/routes/comments.js

```diff
@@ -0,0 +1,18 @@
+import db from "../db/Db-connection.js";
+import express from "express";
+const router = express.Router();
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+router.get('/', async (req, res) => {
+  
+      try {
+        const itemList = await db.any(`SELECT * FROM comment WHERE anime_id=$1`, [anime_id]);
```

Comment:
Tested query on postico and it is working!

Reply:
Based on what you are saying here: https://github.com/dolivas644/AniTracker/pull/23#issuecomment-1319324359

The request is working, but not returning any data. 

1. What do you expect the api to return? What data do you have in the db that should come back?
2. Where is animeId coming from? I don't see you parsing the query params at all so I think you need to do that somewhere from the `req` object?
3. What value for `animeId` is coming in here?


It seems like the problem is either `anime_id` isn't right and so the query isn't filtering what you expect or something is off with your query and even though the `anime_id` is what you expect, it isn't working in the query how you expect it to.

Reply:
I'm actually confused how this code would run at all as it looks like `anime_id` isn't defined so shouldn't this throw an error?